Module_01/2_CSS/README.md

```diff
@@ -0,0 +1,227 @@
+`width` and `height` are set on the content box. The content box contains an element's content, which could be text or child elements.
+
+There are some other properties, `min-width`, `max-width`, `min-height`, `max-height`. They allow you to control an elements height without having set a hard value, this would allow content to grow or shrink, but only to a set value.
+
+## `padding`
+`padding` is the *inner* margin of the CSS box between the content and the border. Setting this can be done on all sides using `padding`, or individual sides using `padding-top`, `padding-bottom`, `padding-left`, and `padding-right` properties.
+
+## `border`
+The `border` wraps an element acting as a border. It sits between the padding and margin. The `border` proptery is set to 0 by default, which is not visible. You can set a thickness, style, color using shorthand, for instance `border: 1px solid red`.
+
+You can also manipulate just one part of the border using properties like `border-top`. Additionally, you can change just on property using properties like `border-color`. If you want to be really specific you can apply both a specific side and property with properites like `border-top-style`.
+
+## `margin`
+The `margin` is outside of the border. It pushes up against other CSS boxes. It behaves similarly to `padding`, in that you can set all sides with `margin`, or indivdual sides with `margin-left`, `margin-right`, `margin-top`, `margin-bottom`
+
+## Try it out
+Try manipulating the boxes in [this sandbox](https://codepen.io/Melizzap/pen/QogaWg?editors=1100) using `padding`, `border`, `margin`, `width`, and `height`. Notice how each property changes how the elements appear.
+
+
+# How does CSS affect HTML?
+We browsers apply CSS rules to a document to affect how they are displayed. A CSS rule is made by:
+- A set of `properties`, which have values set to update how the HTML content is displayed.
+- A `selector`, which selects elements on the page you would like to apply the property values to
+
+These rule are contained in a **stylesheet**
+
+## Try it out!
+Visit this [sandbox](https://codepen.io/Melizzap/pen/XGWgMy?editors=1100)
+
+Try adding these styles to the CSS column
+```
+h1 {
+  background: aqua;
+  text-align: center;
+}
+
+p {
+  border: 2px solid green;
+  color: orange;
+  padding: 20px;
+  text-align: center;
+}
+```
+
+The first rule starts with the `h1` selector, which means that it will apply all the properties list to every element with the `<h1>` tag.
+1. **background** changes the background color of the element
+2. **text-align** centers the text
+
+The second rule starts with the `p` selector
+1. **border** applies a color, width, and pattern to the border of the element
+2. **color** sets the text color
+3. **padding** adds space on the inside of the element
+
+Try adding some more styles in the sandbox. [Here is a good resource for CSS properties.](https://developer.mozilla.org/en-US/docs/Web/CSS)
+
+# Under the Hood
```

Comment:
I think this section could be saved for a module on manipulating the DOM with JS.

Reply:
good call, I can move this out of here... the css module is quite long